ligo/src/base_DAO.mligo

```diff
@@ -26,7 +26,9 @@ let base_DAO_contract
             | M_right (fbp) ->
                 let result =
                   if Tezos.amount > 0tez
-                    then (failwith "FORBIDDEN_XTZ" : return)
+                    then
+                      ([%Michelson ({| { FAILWITH } |} : string * unit -> return)]
+                        ("FORBIDDEN_XTZ", ()) : return)
```

Comment:
Wow, this even compiles in what one would expect in the resulting Michelson, nice.